src/main/java/uk/gov/justice/hmpps/prison/api/resource/BookingResource.java

```diff
@@ -217,6 +219,7 @@ public InmateDetail getOffenderBookingByOffenderNo(@PathVariable("offenderNo") @
         @ApiResponse(responseCode = "500", description = "Unrecoverable error occurred whilst processing request.", content = {@Content(mediaType = "application/json", schema = @Schema(implementation = ErrorResponse.class))})})
     @Operation(summary = "Offender detail.", description = "Offender detail for offenders")
     @PostMapping("/offenders")
+    @SlowReportQuery
```

Comment:
hmmm, isn't this a simple query, and doesn't this need to be on the primary for elastic search or whatever?